app/models/common.py

```diff
@@ -17,9 +18,12 @@ def __init__(self, board_move_split):
 
     @staticmethod
     def position_converter(pos_board: str) -> tuple[int, int]:
-        x_pos_board, y_pos_board = (pos_board[0].upper(), pos_board[1])
+        pos_letter = pos_board[0].upper()
+        pos_numb = pos_board[1]
 
-        x_pos_array = np.where(COLUMN_NAMES == x_pos_board)[0][0]
-        y_pos_array = int(y_pos_board) - 1
+        if not (pos_letter in COLUMN_NAMES and int(pos_numb) in range(1, 9)):
```

Comment:
Let's investgate a bit if we could move this check inside MoveValidator in order to have them all in one central place.